src/components/login/login.component.tsx

```diff
@@ -4,6 +4,7 @@ import { ReactElement } from "react";
 import { createStore } from "redux";
 import { FirebaseService } from "../../commons/firebase.service";
 import { SVGs } from "../../commons/svg.list";
+import { SignInForm } from "../signInForm/SignInForm"; 
```

Comment:
This looks right but the directory name isn't camelCased correctly. eg `src/components/signinForm/SigninForm.js` should be `src/components/signInForm/SignInForm.js`

Reply:
I had made this change in my code editor. I realized the other day that I cloned the repo instead of forking it way back at the first meetup when I first was checking out the code.  I wonder if that's part of my issue?